WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersNavigationController.swift

```diff
@@ -0,0 +1,86 @@
+import UIKit
+
+class BloggingRemindersNavigationController: LightNavigationController {
+
+    private let viewControllerDrawerPositions: [DrawerPosition]
+
+    init(rootViewController: UIViewController, viewControllerDrawerPositions: [DrawerPosition]) {
+        self.viewControllerDrawerPositions = viewControllerDrawerPositions
+
+        super.init(rootViewController: rootViewController)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override public var preferredContentSize: CGSize {
+        set {
+            viewControllers.last?.preferredContentSize = newValue
+            super.preferredContentSize = newValue
+        }
+        get {
+            guard let visibleViewController = viewControllers.last else {
+                return .zero
+            }
+
+            return visibleViewController.preferredContentSize
+        }
+    }
+
+    override func pushViewController(_ viewController: UIViewController, animated: Bool) {
+        super.pushViewController(viewController, animated: animated)
+
+        updateDrawerPosition()
+    }
+
+    override func popViewController(animated: Bool) -> UIViewController? {
+        let viewController = super.popViewController(animated: animated)
+
+        updateDrawerPosition()
+
+        return viewController
+    }
+
+    private func updateDrawerPosition() {
+        let index = max(min(viewControllers.count-1, viewControllerDrawerPositions.count-1), 0)
+        let newPosition = viewControllerDrawerPositions[index]
+
+        if let viewController = viewControllers.last {
+            preferredContentSize = viewController.preferredContentSize
+        }
+        
+        if let bottomSheet = self.parent as? BottomSheetViewController, let presentedVC = bottomSheet.presentedVC {
```

Comment:
I wonder if we could do something different instead of relying on the responder chain? Just in case it changes at a later time, for any reason.

Reply:
Possibly, I took this from the way the prepublishing flow currently works. I'll leave as-is for now, but we can always rework this!